mmeval/metrics/lvis.py

```diff
@@ -0,0 +1,338 @@
+        ...     w = np.random.rand(num_bboxes, ) * (img_w - x)
+        ...     h = np.random.rand(num_bboxes, ) * (img_h - y)
+        ...     return np.stack([x, y, x + w, y + h], axis=1)
+        >>>
+        >>> def _gen_masks(bboxes, img_w=256, img_h=256):
+        ...     if mask_util is None:
+        ...         raise ImportError(
+        ...             'Please try to install official pycocotools by '
+        ...             '"pip install pycocotools"')
+        ...     masks = []
+        ...     for i, bbox in enumerate(bboxes):
+        ...         mask = np.zeros((img_h, img_w))
+        ...         bbox = bbox.astype(np.int32)
+        ...         box_mask = (np.random.rand(
+        ...             bbox[3] - bbox[1],
+        ...             bbox[2] - bbox[0]) > 0.3).astype(np.int32)
+        ...         mask[bbox[1]:bbox[3], bbox[0]:bbox[2]] = box_mask
+        ...         masks.append(
+        ...             mask_util.encode(
+        ...                 np.array(mask[:, :, np.newaxis], order='F',
+        ...                          dtype='uint8'))[0])  # encoded with RLE
+        ...     return masks
+        >>>
+        >>> img_id = 1
+        >>> img_w, img_h = 256, 256
+        >>> num_bboxes = 10
+        >>> pred_boxes = _gen_bboxes(
+        ...     num_bboxes=num_bboxes,
+        ...     img_w=img_w,
+        ...     img_h=img_h)
+        >>> pred_masks = _gen_masks(
+        ...     bboxes=pred_boxes,
+        ...     img_w=img_w,
+        ...     img_h=img_h)
+        >>> prediction = {
+        ...     'img_id': img_id,
+        ...     'bboxes': pred_boxes,
+        ...     'scores': np.random.rand(num_bboxes, ),
+        ...     'labels': np.random.randint(0, num_classes, size=(num_bboxes, )),
+        ...     'masks': pred_masks
+        ... }
+        >>> gt_boxes = _gen_bboxes(
+        ...     num_bboxes=num_bboxes,
+        ...     img_w=img_w,
+        ...     img_h=img_h)
+        >>> gt_masks = _gen_masks(
+        ...     bboxes=pred_boxes,
+        ...     img_w=img_w,
+        ...     img_h=img_h)
+        >>> groundtruth = {
+        ...     'img_id': img_id,
+        ...     'width': img_w,
+        ...     'height': img_h,
+        ...     'bboxes': gt_boxes,
+        ...     'labels': np.random.randint(0, num_classes, size=(num_bboxes, )),
+        ...     'masks': gt_masks,
+        ...     'ignore_flags': np.zeros(num_bboxes)
+        ... }
+        >>> lvis_det_metric(predictions=[prediction, ], groundtruths=[groundtruth, ])  # doctest: +ELLIPSIS  # noqa: E501
+        {'bbox_mAP': ..., 'bbox_mAP_50': ..., ...,
+         'segm_mAP': ..., 'segm_mAP_50': ..., ...,
+         'bbox_result': ..., 'segm_result': ..., ...}
+    """
+
+    def __init__(self,
+                 ann_file: Optional[str] = None,
+                 metric: Union[str, List[str]] = 'bbox',
+                 classwise: bool = False,
+                 proposal_nums: Sequence[int] = (100, 300, 1000),
```

Comment:
![image](https://user-images.githubusercontent.com/48282753/220494157-679fd33a-b177-4086-a817-7441a706ae88.png)

proposal_nums used in 
In lvis, max dets is 300, so we may need to set proposal_nums to 300